src/windows/SliderWindows/SliderView.cpp

```diff
@@ -96,11 +96,9 @@ namespace winrt::SliderWindows::implementation {
                 if (propertyValue.IsNull()) {
                     this->ClearValue(xaml::Controls::Slider::StepFrequencyProperty());
                 }
-                else if (propertyValue.AsDouble() != 0) {
-                    this->StepFrequency(propertyValue.AsDouble());
-                }
                 else {
-                    this->StepFrequency(c_stepDefault);
+                    auto&& step = CalculateStepFrequencyPercentageValue(propertyValue.AsDouble());
```

Comment:
Does assigning to an r-value reference provides any benefit if you don't use std::move?

Reply:
`auto&&` will bind to both r-values and l-values. It is not always and only r-value reference. Scott Meyers in his book used to call it "*universal reference*".
Let me know if I understand your comment correctly.

Reply:
I was imprecise, and std::move is relevant here. My point was that while `auto&&` can bind both l- and r-value refs, what the CalculateStep function returns is an r-value. As such, it'd be enough to just write 
`auto step = ..` since you're just capturing an ephemeral value. So in this case, following Meyer's explanation, `auto&& step` gets resolved to an l-value reference, but I'm not sure this is needed, since this new reference doesn't really refer to anything after the function returns the rvalue.

tl;dr; I just meant you can skip `&&`